SeleniumPracticalExercise/PageObjects/AddEmployeePage.cs

```diff
@@ -17,11 +17,11 @@ public AddEmployeePage(IWebDriver driver) : base(driver)
         }
 
         /// <summary>
-        /// Enter Firstname & Lastname
-        /// then Click Save
-        /// did stale so we can wait for element to save. 
+        /// Enter First and Last name
         /// </summary>
-        public void SetEnterNames(string firstname, string lastname)
+        /// <param name="firstname"></param>
```

Comment:
These docs are empty. You should describe each parameter... I saw this elsewhere as well so make sure you go back and fill all of those in as well. Examples can be found in BasePageLocal.cs.